web/locales/de.json

```diff
@@ -27,6 +27,7 @@
         "back": "zurück",
         "source": "Quelle",
         "fossilfuels": "fossile Brennstoffe",
+        "carbonintensity": "CO2 Konzentration",
```

Comment:
Actually, to correct myself: it should be rather `CO2-Konzentration`. Here and anywhere else.